test/e2e/README.md

```diff
@@ -0,0 +1,16 @@
+# E2E Testing
```

Comment:
Should we add a section to our Development guide on this - at least we should add the new major `Make` targets?

I feel like there should be a "how to run e2e tests" section, or maybe section per deployment? Or something.

Then I can follow the guide, and make sure it all works :smile:

Reply:
yes but I think only make e2e would be sufficient no ? Do you see a need to talk about the e2e cluster ? Since people could just use their development cluster or minikube which is already well documented.

So far how to run e2e is pretty straightforward after you've setup your cluster, just run `make test-e2e`. I can add small sections about how just after talking about `Running a Test Minikube cluster` which would explain how to run against those 2 clusters and include the make target.

Do you think I should explain the e2e image and gcloud-e2e targets which are exclusively for cloud builder, I'm actually not sure that's why I ask ?

Reply:
No, I think you are right - we should document how to run the e2e tests, I don't think we need to document how they are built.

Will need to also have `make minikube-test-e2e` in the minikube section but that's about it.

:+1:

Reply:
I've updated the build/ documentation, let me know.

Reply:
LGTM!